_includes/page_footer.html

```diff
@@ -1,7 +1,7 @@
 <footer class="footer-cockpit">
     <p>Cockpit is free software and released under the
         <a href="http://www.gnu.org/licenses/old-licenses/gpl-2.0.html">LGPL v2.1+</a></p>
-    <p>© Cockpit contributors 2017</p>
+    <p>© {{ site.time | date: "%Y" }} Cockpit contributors</p>
```

Comment:
To be honest I'd just drop that line. From a legal POV this is bogus anyway, as this is not a valid copyright. So if it's a static or dynamically wrong one really doens't matter..